mkdocs/overrides/devbook/transactions/bonded-aggregate.py

```diff
@@ -0,0 +1,261 @@
+import json
+import os
+import time
+import urllib.request
+
+from symbolchain.CryptoTypes import Hash256, PrivateKey
+from symbolchain.facade.SymbolFacade import SymbolFacade
+from symbolchain.sc import Amount
+from symbolchain.symbol.IdGenerator import generate_mosaic_alias_id
+from symbolchain.symbol.Network import NetworkTimestamp
+
+NODE_URL = 'https://001-sai-dual.symboltest.net:3001'
+print(f'Using node {NODE_URL}')
+
+# Helper function to announce transaction
+def announce_transaction(payload, endpoint, label):
+	print(f'Announcing {label} to {endpoint}')
+	request = urllib.request.Request(
+		f'{NODE_URL}{endpoint}',
+		data=payload.encode(),
+		headers={'Content-Type': 'application/json'},
+		method='PUT'
+	)
+	with urllib.request.urlopen(request) as response:
+		print(f'  Response: {response.read().decode()}')
+
+# Helper function to wait for transaction status
+def wait_for_status(hash_value, expected_status, label):
+	print(
+		f'Waiting for {label} to reach {expected_status} status from '
+		f'/transactionStatus/{hash_value}'
+	)
+
+	status_path = f'/transactionStatus/{hash_value}'
+	for attempt in range(60):
+		time.sleep(1)
+		try:
+			with urllib.request.urlopen(
+				f'{NODE_URL}{status_path}'
+			) as response:
+				status = json.loads(response.read().decode())
+				if status['group'] == expected_status:
+					if expected_status == 'confirmed':
+						print(f'  Transaction status: {status["group"]}')
+						print(
+							f'{label} confirmation in ' +
+							f'{attempt + 1} seconds')
+					elif expected_status == 'partial':
+						print(
+							'  Transaction is partial, '
+							'ready for cosignatures'
+						)
+					return attempt + 1
+				if status['group'] == 'failed':
+					print(f'  Transaction status: {status["group"]}')
+					print(f'{label} failed: {status["code"]}')
+					return attempt + 1
```

Comment:
This should throw an exception so the program halts. Same thing with `HTTPError` and timeouts.

Otherwise, we print the error and continue execution, which is confusing.